manifests/m/MikeFarah/yq/4.35.2/MikeFarah.yq.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: MikeFarah.yq
+PackageVersion: 4.35.2
+InstallerLocale: en-US
+InstallerType: portable
+Commands:
+- yq
+ReleaseDate: 2023-09-26
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/mikefarah/yq/releases/download/v4.35.2/yq_windows_amd64.exe
+  InstallerSha256: 2E10B644B2FDF7D62354487A075E43FCD7D2F32B972DF713AD5855FD2581EB16
+- Architecture: x64

```

Comment:
```suggestion
- Architecture: x64
  InstallerUrl: https://github.com/mikefarah/yq/releases/download/v4.35.2/yq_windows_amd64.exe
  InstallerSha256: 2E10B644B2FDF7D62354487A075E43FCD7D2F32B972DF713AD5855FD2581EB16
- Architecture: x86
```

Reply:
Hi @Okeanos,

It looks like the architectures might have become swapped.

Reply:
D'oh … thanks a lot for pointing that out. Should be fixed now 🤞